specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ServerConfigurationOptions.json

```diff
@@ -0,0 +1,264 @@
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets a list of managed instance server configuration options.": {
+            "$ref": "./examples/ServerConfigurationOptionList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/serverConfigurationOptions/{serverConfigurationOptionName}": {
+      "get": {
+        "tags": [
+          "ServerConfigurationOptions"
+        ],
+        "description": "Gets managed instance server configuration option.",
+        "operationId": "ServerConfigurationOptions_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "name": "serverConfigurationOptionName",
+            "in": "path",
+            "description": "The name of the server configuration option.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "allowPolybaseExport"
+            ],
+            "x-ms-enum": {
+              "name": "ServerConfigurationOptionName",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified managed instance server configuration option.",
+            "schema": {
+              "$ref": "#/definitions/ServerConfigurationOption"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidServerConfigurationOptionName - User specified server configuration option name that is not supported.\n\n * 400 InvalidServerConfigurationOptionValue - Server configuration option value \"{0}\" is not supported for server configuration option \"{1}\".\n\n * 400 InvalidServerConfigurationOptionProperties - The provided set of parameters in the body of the request is either empty or invalid.\n\n * 404 SubscriptionNotFound - The requested subscription was not found.\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-examples": {
+          "Gets managed instance server configuration option.": {
+            "$ref": "./examples/ServerConfigurationOptionGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ServerConfigurationOptions"
+        ],
+        "description": "Updates managed instance server configuration option.",
+        "operationId": "ServerConfigurationOptions_CreateOrUpdate",
```

Comment:
>CreateOrUpdate [](http://example.com/codeflow?start=51&length=14)

CreateOrUpdate OR Only Update?

Reply:
Well, currently it will only do update (create is not possible as I explained in other comment). I left CreateOrUpdate in name though as per my understanding that is the standard naming when using the PUT operation and we want to use PUT here.